src/ReactiveUI.XamForms.Tests/Activation/Mocks/MockForms.cs

```diff
@@ -0,0 +1,241 @@
+﻿// Copyright (c) 2021 .NET Foundation and Contributors. All rights reserved.
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+
+namespace ReactiveUI.Tests
+{
+    /// <summary>
+    /// Mock Forms.
+    /// </summary>
+    public static class MockForms
```

Comment:
Why not just use [Xamarin.Forms.Mocks](https://github.com/jonathanpeppers/Xamarin.Forms.Mocks)

Reply:
Thanks I will update to this, I didn't realise this existed